AAUnlimited/MemMods/AAPlay/Events/ScreenCapture.cpp

```diff
@@ -68,6 +68,10 @@ namespace PlayInjections {
 			wcsncpy_s(path + pathLength - 3, 4, ext, 3);
 			bitmap->Save(path, cls, params);
 			delete bitmap;
+
+			char notify_text[256];
+			sprintf(notify_text, "Screenshot saved (%ws)", path);
```

Comment:
This is unsafe. Use `sprintf_s` instead. But not really, use `std::wstring` instead.

Reply:
Ok